content/recipes/authoring/kata-snippets/reference-solution.md

```diff
@@ -16,6 +16,6 @@ Failure to comply with the below guidelines should be considered an issue to be
 
 ## General Guidelines
 
-- **Conform to [General Coding Guidelines](/recipes/authoring/kata-snippets/coding-general/): reference solution is code too, and, as such, should keep up to code quality standards. It will make maintenance of the kata much easier, and readability of reference solution can be very helpful for future maintarners and translators. Reference solution is not for you to write, but it's for others to read.
+- **Conform to [General Coding Guidelines](/recipes/authoring/kata-snippets/coding-general/)**: reference solution is code too, and, as such, should keep up to code quality standards. It will make maintenance of the kata much easier, and readability of reference solution can be very helpful for future maintainers and translators. Reference solution is not for you to write, but for others to read.
```

Comment:
```suggestion
- **Conform to [General Coding Guidelines](/recipes/authoring/kata-snippets/coding-general/)**: the reference solution is code too, and, as such, should keep up to code quality standards. It will make maintenance of the kata much easier, and the readability of the reference solution can be very helpful for future maintainers and translators. The reference solution is not for you to write, but for others to read.
```